lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -114,3 +124,40 @@ func ProductVersion() string {
 func VchConfig() *config.VirtualContainerHostConfigSpec {
 	return vchConfig
 }
+
+func createImageStore() error {
+
+	// TODO(jzt): we should move this to a utility package or something
+	host, err := sys.UUID()
+	if host == "" {
+		host, err = os.Hostname()
+	}
```

Comment:
I do run nested, but I believe this pattern came from imagec - you may be right, it may be deprecated. Perhaps it was so that imagec could run in standalone mode (in which case, we should probably still keep it).